### DISCUSSION TITLE
Why does it automatically adds typescript to devDependencies

### DISCUSSION THREAD

**Discussion by khokm** [parcel-bundler/parcel#5265] [General] [answered]
Why does it automatically adds typescript to devDependencies, even if typescript with same version already installed globally?

**Comment by DeMoorJasper** [comment, marked as answer]
Not sure why we do that but it is highly recommended to have compilers locally installed in a project regardless so any collaborators on the project are automatically using the same version as you to prevent conflicts.

**Reply by mischnic** [reply]
(And a global version cannot be imported just with `require("typescript")` in a project anyway.)
